src/lib/components/composites/settings/user-settings/ShortcutsVisibilitySettings.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+    import { Button } from "$lib/components/primitives/button";
+    import { Checkbox } from "$lib/components/primitives/checkbox";
+    import { Label } from "$lib/components/primitives/label";
+    import { shortcuts } from "$lib/global-state/shortcuts-visibility-state.svelte";
+    import SettingsSection from "../SettingsSection.svelte";
+</script>
+
+<!--
+@component
+Component for changing the shortcuts visibility setting.
+
+Usage:
+```svelte
+    <ShortcutsVisibilitySettings />
+```
+-->
+<SettingsSection sectionTitle="Shortcuts Visibility">
+    <div class="flex-start flex space-x-2">
+        <Checkbox id="shortcuts-visibility-checkbox" bind:checked={shortcuts.isVisible} />
+        <div class="grid gap-1.5 pt-0.25 leading-none">
```

Comment:
`leading-none` doesn't seem to make a difference.
```suggestion
        <div class="grid gap-1.5 pt-0.25">
```

Reply:
You're right. I also removed `flex-start` because it also wasn't needed.